### ISSUE TITLE
As a user, I only want to receive 1 match email per day, so I dont unsubscribe

### ISSUE THREAD

**Issue by owocki** [gitcoinco/web#454]
# the problem

![screen shot 2018-02-15 at 7 37 13 am](https://user-images.githubusercontent.com/513929/36261967-55f11bd8-1223-11e8-94af-7bf407a62234.png)

too many issues posted to the platform, not enough room in my inbox!

# the solution

* refactor the code that sends this email to be kicked off once daily, via a crontab, instead of triggered when a new issue is created
* add checks to make sure that the email isnt sent more than 1x a day.

**Comment by mkosowsk** [comment]
Counterpoint, I like to drink from the firehose and have a lot of emails... 

I think I'm in the minority but would appreciate to still have this option!

**Comment by gitcoinbot** [comment]
__This issue now has a funding of 0.03 ETH (27.31 USD) attached to it.__

 * If you would like to work on this issue you can claim it [here](https://gitcoin.co/funding/details?url=https://github.com/gitcoinco/web/issues/454).
 * If you've completed this issue and want to claim the bounty you can do so [here](https://gitcoin.co/funding/details?url=https://github.com/gitcoinco/web/issues/454)
 * Questions? Get help on the <a href='https://gitcoin.co/slack'>Gitcoin Slack</a>
 * $9550.92 more Funded OSS Work Available at: https://gitcoin.co/explorer

**Comment by owocki** [comment]
@mkosowsk maybe its worth making this configurable in ones email settings?

**Comment by mkosowsk** [comment]
@owocki is the idea that one email would have multiple issues collected over a 24 hour time frame (if applicable)?

If that's the case, I'd be happy with that!

**Comment by owocki** [comment]
that would be hot!  a rollup email with all of the funded work available thats a match for me would be 💯

**Comment by gitcoinbot** [comment]
__Work has been started on the 0.03 ETH (28.02 USD) funding by__: 
 1. [@gsvitak](https://gitcoin.co/profile/gsvitak) 

 __Please work together__ and coordinate delivery of the issue scope. Gitcoin doesn't know enough about everyones skillsets / free time to say who should work on what, but we trust that the community is smart and well-intentioned enough to work together.  As a general rule; if you start work first, youll be at the top of the above list ^^, and should have 'dibs' as long as you follow through. 

 On the above list? Please leave a comment to let the funder (@owocki) and the other parties involved what you're working, with respect to this issue and your plans to resolve it.  If you don't leave a comment, the funder may expire your submission at their discretion. 

 * Learn more [on the gitcoin issue page](https://gitcoin.co/funding/details?url=https://github.com/gitcoinco/web/issues/454)
 * Questions? Get help on the <a href='https://gitcoin.co/slack'>Gitcoin Slack</a>
 * $12987.58 more Funded OSS Work Available at: https://gitcoin.co/explorer

**Comment by gsvitak** [comment]
@mkosowsk have you already started work on this feature?

**Comment by mkosowsk** [comment]
@gsvitak I have not started work on this, it's all yours!

**Comment by gitcoinbot** [comment]
__Work has been started on the 0.03 ETH (25.67 USD) funding by__: 
 1. [@gsvitak](https://gitcoin.co/profile/gsvitak) 

 __Please work together__ and coordinate delivery of the issue scope. Gitcoin doesn't know enough about everyones skillsets / free time to say who should work on what, but we trust that the community is smart and well-intentioned enough to work together.  As a general rule; if you start work first, youll be at the top of the above list ^^, and should have 'dibs' as long as you follow through. 

 On the above list? Please leave a comment to let the funder (@owocki) and the other parties involved what you're working, with respect to this issue and your plans to resolve it.  If you don't leave a comment, the funder may expire your submission at their discretion. 

 * Learn more [on the gitcoin issue page](https://gitcoin.co/funding/details?url=https://github.com/gitcoinco/web/issues/454)
 * Questions? Get help on the <a href='https://gitcoin.co/slack'>Gitcoin Slack</a>
 * $9355.57 more Funded OSS Work Available at: https://gitcoin.co/explorer

**Comment by owocki** [comment]
hi from gitcoin.co here ==> was just debugging an issue with gitcoinbot leaving too many comments about work starting.. i put in a fix and we should be okay now.

**Comment by gsvitak** [comment]
Status update. I was blocked by a 404 error when I clicked on the Email Settings link. The slack thread https://gitcoincommunity.slack.com/archives/C8E45J5D0/p1519619010000007 helped me get past the block.

**Comment by mkosowsk** [comment]
Documenting this now with screen shots as eventual that Slack thread will disappear down the Slack hole 👍 

<img width="861" alt="screen shot 2018-02-26 at 10 27 55 am" src="https://user-images.githubusercontent.com/7970539/36678820-c380c134-1adf-11e8-984e-6887c2a76c5b.png">

<img width="849" alt="screen shot 2018-02-26 at 10 28 06 am" src="https://user-images.githubusercontent.com/7970539/36678823-c4fcd4b2-1adf-11e8-9605-4b6e44c415b5.png">

**Comment by PixelantDesign** [comment]
Is design needed?

**Comment by owocki** [comment]
i think we could stand to redesign almost all of our emails... as most of them were coded during my "im getting up at 3am to work on gitcoin while i have a full time job" phase of the project..

but that aside... if you could spend 1/2 hour mocking up what this should look like... it might be worthwhile.  here are two templates to jumpstart the visual conversation http://bits.owocki.com/0j0B2z0M0m3Q/Archive%202018-02-27%20at%208.49.37%20AM.zip

**Comment by mbeacom** [comment]
@owocki @PixelantDesign I spoke with @gsvitak regarding this issue and tried to provide him with some clarification on this bounty.  Will we want to redesign the new bounty template before moving forward with this?

**Comment by owocki** [comment]
i wonder if we couldnt get away with just modifying the following template

![screencapture-localhost-8080-_administration-email-new_bounty-1520373574894](https://user-images.githubusercontent.com/513929/37060969-20450bf8-214f-11e8-933b-539d8e03a786.png)

as follows:

![screencapture-localhost-8080-_administration-email-new_bounty-1520373619507](https://user-images.githubusercontent.com/513929/37060972-23e482d4-214f-11e8-894a-1529add34c22.png)

cc @PixelantDesign in case she has some thoughts here

**Comment by gsvitak** [comment]
k I will move forward with the design above unless @PixelantDesign has some ideas.

**Comment by PixelantDesign** [comment]
@gsvitak Here is an update email template. Let me know if you think any of this is not doable, I tried to keep it pretty simple. I've removed a few fields to clean it up a bit @owocki.

![gitcoin_newissues_email](https://user-images.githubusercontent.com/238235/37111464-bcfacece-220d-11e8-8b53-66a73e949e59.png)

Fonts I used are Futura which is already in the system and Muli Regular and Semibold which can be found [here](https://fonts.google.com/specimen/Muli)

**Comment by owocki** [comment]
this email template is 👌 !  we should consider moving all emails to this template

**Comment by owocki** [comment]
(thinking out loud here) is the gitcoin logo too big / billboardey on this?

**Comment by PixelantDesign** [comment]
@owocki That's modern email design! :P Might be the combination of knocked out logo and high contrast. We could shrink it about 25%.

**Comment by mbeacom** [comment]
👍 for MJML 😁

**Comment by mbeacom** [comment]
@DMXSF1 Were you still interested in working on this issue?

**Comment by owocki** [comment]
im going to wrap this up now

**Comment by gitcoinbot** [comment]
@dmxsf1 are you still working on this issue?

**Comment by gitcoinbot** [comment]
@dmxsf1 are you still working on this issue?

**Comment by gitcoinbot** [comment]
@dmxsf1 are you still working on this issue?

**Comment by owocki** [comment]
im gonna close this issue now

**Comment by gitcoinbot** [comment]
__The funding of 0.03 ETH (19.35 USD @ $645.03/ETH) attached to this issue has been **killed** by the bounty submitter__

 * Questions? Get help on the <a href='https://gitcoin.co/slack'>Gitcoin Slack</a>
 * $6347.99 more Funded OSS Work Available at: https://gitcoin.co/explorer

**Comment by weberV** [comment]
Issue Status: 1. Open 2. **Expired** 

<hr>

__This issue now has a funding of 0.03 ETH (20.76 USD @ $692.1/ETH) attached to it.__

 * If you would like to work on this issue you can 'start work' [on the Gitcoin Issue Details page](http://localhost:8000/issue/gitcoinco/web/454/47).
 * Questions? Checkout <a href='https://gitcoin.co/help'>Gitcoin Help</a> or the <a href='https://gitcoin.co/slack'>Gitcoin Slack</a>
 * $896.27 more funded OSS Work available on the [Gitcoin Issue Explorer](https://gitcoin.co/explorer)
